lms/djangoapps/courseware/access_utils.py

```diff
@@ -105,7 +105,19 @@ def enterprise_learner_enrolled(request, user, course_key):
         enterprise_customer_user__user_id=user.id,
         enterprise_customer_user__enterprise_customer__uuid=enterprise_customer_data['uuid'],
     )
-    return enterprise_enrollments.exists()
+    enterprise_enrollment_exists = enterprise_enrollments.exists()
+    log.debug(
```

Comment:
For any concerns around over-logging here, I specifically made sure to emit this log after checking for `user.is_authenticated` and `learner_portal_enabled` so this should hopefully keep the messages down to the minimal set I need for testing.

Reply:
probably want to switch to `log.info()`, right?